server/plugin.go

```diff
@@ -204,6 +208,79 @@ func (p *Plugin) OnActivate() error {
 	}
 
 	go p.initStats()
+
+	go func() {
+		time.Sleep(time.Second * 10)
+
+		instances, err := p.instanceStore.LoadKnownJIRAInstances()
+		if err != nil {
+			p.API.LogError("unable to register autolinks", "err", err)
+			return
+		}
+
+		for url := range instances {
+			instance, err := p.instanceStore.LoadJIRAInstance(url)
+			if err != nil {
+				continue
+			}
+
+			jci, ok := instance.(*jiraCloudInstance)
+			if !ok {
+				p.API.LogWarn("only cloud instances supported for autolink", "err", err)
+				continue
+			}
+
+			if err := p.AddAutolinksForCloudInstance(jci); err != nil {
+				p.API.LogWarn("could not install autolinks for cloud instance", "instance", jci.BaseURL, "err", err)
+				continue
+			}
+		}
+	}()
+
+	return nil
+}
+
+func (p *Plugin) AddAutolinksForCloudInstance(jci *jiraCloudInstance) error {
+	client, err := jci.getJIRAClientForServer()
+	if err != nil {
+		return fmt.Errorf("unable to get jira client for server: %w", err)
+	}
+
+	keys, err := JiraClient{Jira: client}.GetAllProjectKeys()
+	if err != nil {
+		return fmt.Errorf("unable to make jira client: %w", err)
```

Comment:
I think this makes more sense
```suggestion
		return fmt.Errorf("unable to get project keys: %w", err)
```